pkgs/data/fonts/cozette/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, mkfontscale }:
+
+let
+  version = "1.5.1";
+  releaseUrl =
+    "https://github.com/slavfox/Cozette/releases/download/v.${version}";
+in stdenv.mkDerivation rec {
```

Comment:
Looks like you clobbered the `pname` and `version` attributes. Please add them back. Since you `let`-bind `version`, I believe you can just use an `inherit version;` instead of `version = version;`.

Reply:
Okay.